apps/api/src/app/workflows-v2/util/url-utils.ts

```diff
@@ -0,0 +1,6 @@
+export function isValidUrlForActionButton(url: string): boolean {
+  const fullUrlPattern = /^(https?:\/\/[^\s/$.?#].[^\s]*)$/i;
+  const partialPathPattern = /^(\/[^\s]*)?$/;
```

Comment:
@LetItRock @SokratisVidros, Do we want to limit the relative URL paths to starting with "/" or should they be any strings?